packages/flutter_tools/test/commands.shard/permeable/upgrade_test.dart

```diff
@@ -238,16 +243,17 @@ void main() {
 
       testUsingContext('upgrade continue prints welcome message', () async {
         final UpgradeCommand upgradeCommand = UpgradeCommand(fakeCommandRunner);
-        await runner.run(
+        applyMocksToCommand(upgradeCommand);
+
+        await createTestCommandRunner(upgradeCommand).run(
```

Comment:
Without the test command runner, this was actually invoking io.exit

Reply:
Great catch!

Reply:
99% @grouma